src/applications/disability-benefits/526EZ/config/form.js

```diff
@@ -496,10 +498,17 @@ const formConfig = {
                   'ui:options': {
                     labels: {
                       yes: 'Yes',
-                      no: 'No, please get them from my doctor'
+                      no: 'No, my doctor has my medical records'
```

Comment:
Should these be booleans?

Reply:
I don't think so. Looks like we usually use y/n as keys in other forms. I think booleans are reserved words so probably can't be used as keys in any case.